### DISCUSSION TITLE
Does the repo currently build?

### DISCUSSION THREAD

**Discussion by JakeSays** [microsoft/Extensible-Storage-Engine#16] [General]
I noticed some recent activity around build support. Is it possible to get a functional build yet?

**Comment by 2BitSalute** [comment]
Yes, the core project (ese.dll) is building, and the buildable and runnable unit tests are also included. Check out [BUILDING.md](https://github.com/microsoft/Extensible-Storage-Engine/blob/main/BUILDING.md).

I haven't updated CONTRIBUTING.md to say that some build and test support is there. I may do it sooner rather than later, or wait until I add the comments. Otherwise, applying patches is difficult.

**Reply by JakeSays** [reply]
By comments do you mean code comments? Because that would be very helpful!

**Comment by MsftBrettShirley** [comment]
Comments are now added as well.  Before any sync out of internal Microsoft ESE changes to github, I re-test the cmake build system.
